pdfminer/pdfdocument.py

```diff
@@ -64,6 +64,10 @@ class PDFTextExtractionNotAllowed(PDFEncryptionError):
     pass
 
 
+class PDFPasswordIncorrect(PDFEncryptionError):
+    pass
```

Comment:
This is needed, as we want to continue using the current error exception https://github.com/splunk-soar-connectors/parser/blob/next/parser_methods.py#L432

Once this PR https://github.com/pdfminer/pdfminer.six/pull/706 gets merged to the main repo, we can update our app and remove this.